docs/modules/ROOT/pages/set-budget.adoc

```diff
@@ -0,0 +1,37 @@
+= Setting a Budget
+:description: Use the Budget Tracker to help you to monitor and control your spending in Hazelcast Viridian.
+:cloud-tags: Manage Accounts
+:cloud-title: Setting a Budget
+:cloud-order: 58
+
+{description}
+
+Set a budget and you'll receive an email alert when your costs reach 80% of your budget, and again when you hit your budget. You can use this information to take action to limit your spending or to gain insights into your resource usage.
+
+== Setting Your Budget
+
+You can set a daily and/or monthly budget.
+
+- Daily budgets: Track your spending from 12 AM (UTC-0) each day.
+- Monthly budgets: Track your spending from the first day of each calendar month. This means that if, for example, you set your budget halfway through a month, your costs are still calculated from the start of the month and compared against your budget for the entire month. Your budget and spending are reset at the start of the next calendar month.
+
+Budgets are set per account and include the cost of resources used by all clusters attached to your account. 
+
+. Sign into the link:{page-cloud-console}[Hazelcast {hazelcast-cloud} console,window=_blank].
+. Go to *Account* > *Billing & Payments*.
+. In the *Budget Tracker*, click on an edit icon.
+. In the *Daily Alert* box, enter your daily budget. Or, set the toggle to *OFF* to only set a monthly budget.
+. In the *Monthly Alert* box, enter your monthly budget. Or, set the toggle to *OFF* to only set a monthly budget.
```

Comment:
Please note, there is also a validation that doesn't allow Monthly Alert to be less than Daily Alert. Perhaps, this information can be included as well.

Reply:
As there is validation, I don't think we need to include in the docs.